enterprise/views.py

```diff
@@ -98,6 +106,34 @@ def get_global_context(request):
         'header_logo_alt_text': _('{platform_name} home page').format(platform_name=platform_name),
     }
 
+def get_localized_price_text(price, request):
```

Comment:
Seems like this would be better in edx-platform or some other python package where it could be reused.

Reply:
Agreed - probably with edx-platform since that is where the middleware that sets country_code lives.